iaso/api/org_units.py

```diff
@@ -615,17 +612,13 @@ def create_org_unit(self, request):
         org_unit_type = get_object_or_404(OrgUnitType, id=org_unit_type_id)
         org_unit.org_unit_type = org_unit_type
 
-        if reference_instance_id and org_unit_type:
-            instance = Instance.objects.get(pk=reference_instance_id)
-            # Check if the instance has as form the reference_form for the orgUnittype
-            # if the reference_form is the same as the form related to the instance one,
-            # assign the instance to the orgUnit as a reference instance
-            if org_unit_type.reference_form == instance.form:
-                org_unit.reference_instance = instance
-
         org_unit.save()
         org_unit.groups.set(new_groups)
 
+        if reference_instance_id and org_unit_type:
```

Comment:
When creating an Org Unit, shouldn't we be allowed to pass a table of `reference_instance_ids`?

Reply:
I asked myself this question, but it would have required too many changes on the front-end.

And since the front-end is only flagging references one by one, I sticked with one `reference_instance_ids`…